modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/BashClientCodegen.java

```diff
@@ -571,16 +573,17 @@ public CodegenOperation fromOperation(String path, String httpMethod,
         List codesamples = (List)op.vendorExtensions.get("x-code-samples");
 
         for (Object codesample : codesamples) {
-          ObjectNode codesample_object = (ObjectNode)codesample;
+            if(codesample instanceof ObjectNode) {
+                ObjectNode codesample_object = (ObjectNode) codesample;
 
-          if((codesample_object.get("lang").asText()).equals("Shell")) {
+                if ((codesample_object.get("lang").asText()).equals("Shell")) {
```

Comment:
TBH I have no idea what this code is supposed to do, but I do know it's an unsafe type cast.  So... I'm making it safe, but still don't know why this is here.